### PR TITLE
Feature/search 1620 docker ssl

### PR DESCRIPTION
**Pull request by aborroy**
Including also SEARCH-1621 fix.

### CODE REVIEW

**Comment by meenalbhave** [comment]
Thanks, @aborroy, The changes for 1620, 1621 and the documentation / read me provided look good.
As we discussed:
1. retain the contents of /6.x < .env and docker-compose.yml> for now, as these are being used by all our build plans and allow easy setup of acs-solr without ssl. .env versions may need fixing as you proposed.
2. SearchServices is community code, so rename community to say ssl (for all ssl setup related files)
3. Move contents of enterprise + ssl-tool to insight engine project on Gitlab so it's easy to setup insight engine with ssl too.
Makes sense?

File: packaging/src/docker/6.x/.env [hunk 1]
```diff
@@ -1,6 +1,5 @@
-# In order to start the community edition set the value of ALFRESCO_EDITION to -community
-ALFRESCO_EDITION=
```
Comment (meenalbhave) [inline packaging/src/docker/6.x/.env]: Can the ALFRESCO_EDITION parameter be put back please?

File: packaging/src/docker/6.x/docker-compose.yml [hunk 2]
```diff
@@ -1,56 +1,82 @@
...
-  activemq:
-    image: alfresco/alfresco-activemq:5.15.6
-    ports:
-      - 8161:8161 # Web Console
-      - 5672:5672 # AMQP
-      - 61616:61616 # OpenWire
-      - 61613:61613 # STOMP
\ No newline at end of file
+    alfresco:
+        image: alfresco/alfresco-content-repository-community:${ALFRESCO_TAG}
```
Comment (meenalbhave) [inline packaging/src/docker/6.x/docker-compose.yml]: Change to: alfresco/alfresco-content-repository${ALFRESCO_EDITION}:${ALFRESCO_TAG} please
Insight Engine (sql support) is to be run with Enterprise edition.

File: packaging/src/docker/6.x/.env (hunk 1 above)
Comment (aborroy) [reply packaging/src/docker/6.x/.env]: From ACS 6.1 Docker Compose is not valid, as it requires Transform Services to work (Tika, LibreOffice...). Is this required for some build in Bamboo?

File: packaging/src/docker/6.x/docker-compose.yml (hunk 2 above)
Comment (aborroy) [reply packaging/src/docker/6.x/docker-compose.yml]: The same from above.

**Pull request merged**